.circleci/config.yml

```diff
@@ -55,6 +55,14 @@ jobs:
             K8S_NAMESPACE: hmcts-complaints-formbuilder-adapter-staging
           command: './deploy-scripts/bin/deploy'
       - slack/status: *slack_status
+      - run:
+          name: deploy to staging (EKS cluster)
+          environment:
+            APPLICATION_NAME: hmcts-complaints-formbuilder-adapter
+            PLATFORM_ENV: staging
+            K8S_NAMESPACE: hmcts-complaints-formbuilder-adapter-staging
+          command: './deploy-scripts/bin/deploy-eks'
+      - slack/status: *slack_status
```

Comment:
`slack/status: *slack_status` is always the last thing to happen in a workflow series of steps. it covers any of the run commands above failing. so the one on line 65 is correct but we can therefore remove the one on line 57

Reply:
all done!